docs/user/index.rst

```diff
@@ -1,6 +1,63 @@
 Read the Docs: Documentation Simplified
 =======================================
 
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: Tutorials 🚀
+
+   /tutorial/index
+   /intro/getting-started-with-sphinx
+   /intro/getting-started-with-mkdocs
+   /intro/import-guide
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: How-to guides 🪄
+
+   /guides/authors
+   /guides/administrators
+   /guides/developers
+   Troubleshooting </guides/troubleshooting/index>
+   /examples
+   /faq
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: Explanation 💡
+
+   /choosing-a-site
+   /integrations
+   /pull-requests
+   /science
+   /automatic-redirects
+   /guides/technical-docs-seo-guide
+   /localization
+   /build-notifications
+   /custom-domains
+   /connected-accounts
+   /downloadable-documentation
+   /subprojects
+   /single_version
+   /environment-variables
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: Reference 📚
+
+   /reference/features
+   /about/index
+   /api/index
+   /server-side-search/api
+   /server-side-search/syntax
+   /reference/environment-variables
```

Comment:
Little confused how this one got here? Seems out of place, like it should be under `features`?

Reply:
If we're gonna put anything like this in the top-level, config file is the obvious one to start with, I think.